tests/test_integration/test_partner_resource.py

```diff
@@ -0,0 +1,35 @@
+from tests.test_integration.helpers import create_partner
+from tests.test_integration.test_case import YolaServiceTestCase
+
+
+class TestYolaPartners(YolaServiceTestCase):
+    """Yola: Partner resource"""
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestYolaPartners, cls).setUpClass()
+        cls.partner = create_partner(cls.service)
+        cls.partner_id = cls.partner['id']
+        # There's no way to really delete the test partners we're creating :(
+
+    def test_can_create_partner(self):
```

Comment:
Why the repeated use of `_can`? When looking at the test output it looks a bit like noise/useless word.